lib/utils/zero_hashes.ex

```diff
@@ -0,0 +1,149 @@
+defmodule LambdaEthereumConsensus.Utils.ZeroHashes do
+  @moduledoc """
+  Precomputed zero hashes
+  """
+
+  alias LambdaEthereumConsensus.SszEx
+
+  @bits_per_byte 8
+  @bytes_per_chunk 32
+  @max_merkle_tree_depth 64
+  @zero_hashes <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
```suggestion
    @buffer <<0::size(@bytes_per_chunk * @max_merkle_tree_depth * @bits_per_byte)>>

  @zero_hashes 0..(@max_merkle_tree_depth - 2)
               |> Enum.reduce(@buffer, fn index, acc_buffer ->
                 start = index * @bytes_per_chunk
                 stop = (index + 2) * @bytes_per_chunk
                 focus = acc_buffer |> :binary.part(start, stop - start)
                 left = focus |> :binary.part(0, @bytes_per_chunk)
                 hash = SszEx.hash_nodes(left, left)
                 change_index = (index + 1) * @bytes_per_chunk
                 SszEx.replace_chunk(acc_buffer, change_index, hash)
               end)
```

This way you can compute it at compile time

Reply:
Isn't it faster to have as a constant than to compute this at compile time?

I added the function so @zero_hashes doesn't seem like a magic value.

Reply:
What matters is that it will be equally fast in runtime. I think it's much clearer to have it like this over a hardcoded value. The difference in compile time should be unnoticeable

Reply:
Okay. Updated it